.github/workflows/ci_cd.yml

```diff
@@ -168,6 +168,18 @@ jobs:
             cname: ${{ env.CNAME }}
             token: ${{ secrets.GITHUB_TOKEN }}
 
+  doc-deploy-to-repo:
+    name: "Deploy developers documentation to specific repository"
+    runs-on: ubuntu-latest
+    needs: doc-deploy-dev
+    steps:
+      - name: "Deploy developers documentation"
+        if: github.event_name == 'push'
+        uses: pyansys/actions/doc-deploy-to-repo@main
+        with:
+            cname: ${{ env.CNAME }}
+            repository: "pyansys/actions"
```

Comment:
Could we use this pipeline?:

doc-deploy-to-repo -> doc-deploy-dev -> doc-deploy-stable

The two first are redundant but allow us to check both actions.

Reply:
Oh sure! I can change the order, no problem.